examples/toolkit-demo/candlestick.rb

```diff
@@ -1,5 +1,5 @@
 # ruby candlestick.rb postgres://user:pass@host:port/db_name
-# @see https://jonatas.github.io/timescaledb/candlestick_tutorial
+# @see https://timescale.github.io/timescaledb-ruby/toolkit_candlestick/
```

Comment:
I wasn't sure if this was correct, as I'm not sure there ever was a documented called `candlestick_tutorial` in the repo. :shrug:

Reply:
Yes! it's correct!

Thank you very much! I removed almost all references, but some of the refs in the code are still there!